### DISCUSSION TITLE
Image files not displaying at all!

### DISCUSSION THREAD

**Discussion by vaishaliv20** [community/community#77343] [New to GitHub] [answered]
I am unable to view any of my image files/pdf/ppt after uploading them. Tried everything from changing browser, network to system repair, but still nothing is working. Although I am able to upload my jupyter notebooks easily. Please help. Its super urgent. My job depends on it.

**Comment by ghost** [comment]
like send some screenshots for better understandability

**Comment by leeringrose** [comment]
@vaishaliv20 If you are still struggling with this issue, it would be better to share the screenshots you are facing, at least the link to your repository.

**Comment by vaishaliv20** [comment]
Here is the screenshot. Even after uploading, the image is not displaying and not even able to download raw file.
![github_image_ssue](https://github.com/community/community/assets/90524268/9eb035cd-22b2-49d2-85db-a2071601612b)

**Comment by ghost** [comment, marked as answer]
There could be several reasons why your image files, PDFs, or PPTs are not showing up in GitHub. Here are some possible solutions:

1. **File Size**: GitHub has a strict file size limit of 100MB. If your file is larger than this, it won't upload properly.

2. **File Path**: Make sure the file path in your code matches the actual file path in your repository.

3. **Case Sensitivity**: GitHub is case sensitive, so ensure the case of your file name matches exactly with the one in your code.

4. **File Format**: Ensure the file format is supported by GitHub. For images, formats like JPG, PNG, GIF, and SVG are supported.

5. **Commit and Push**: Make sure you've committed and pushed the changes to your repository.

6. **Check the Raw Button**: If you're trying to view the file on GitHub's website, try clicking the "Raw" button. This will give you a direct link to the file.

7. **.gitattributes File**: Add `*.pdf binary` or `*.ppt binary` to your `.gitattributes` file so these files are treated as binary files.

Remember, these are just some of the common issues and their solutions. Your specific problem might be different. If you're still having trouble, please provide more details about your issue.

**Reply by queenofcorgis** [reply]
Hi @Yuheshpandian , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Reply by ghost** [reply]
Sorry for the mistake I didn't checkout the code of conduct properly but I did now. I would never repeat this. Thank you for the correction

**Reply by FahimS** [reply]
![image](https://github.com/user-attachments/assets/4685ff1b-7c95-4f80-a3e4-b0ec7a768fcd)
![image](https://github.com/user-attachments/assets/cbf66a45-5300-425c-ad36-ad1bfd961a84)

In main branch image shows but in live not show, where is the problem?

**Reply by NavMohan-24** [reply]
I am also facing same issue

**Reply by mikkowsx** [reply]
I found [This](https://blog.jongallant.com/2021/02/png-files-not-rendering-on-github/) and it solved my problem with PNG images.

**Comment by ghost** [comment]
![Screenshot 2023-11-30 191343](https://github.com/community/community/assets/121353576/dcbcf6e5-6554-4cff-8720-b97a8fc1efe6)

**But I could see you files**

**Reply by vaishaliv20** [reply]
Thank you for your help. I will surely follow these steps. You are able to view it, then may be I am missing something.

**Reply by ghost** [reply]
can you mark mine as a answer then it would help me

**Comment by ghost** [comment]
@vaishaliv20 I checked some of you files like .png and .ipynb I could see them very well. So if you want others to see it , there is no problem. But if you want to view them by yourself you need follow the above measures.

**Comment by goyalyashpal** [comment]
hey @vaishaliv20 @queenofcorgis  @leeringrose 
* i am facing the same issue from several months.
*  i am not able to view 
any raw file from github (png , jpg, pdf, svg)
in _any_ repository,
with _any_ site (directly github.com, or in readme files via pypi, via open-vsx, via marketplace.visualstudio.com, etc.) 
from _any_ software (browser, vscodium, vscode, ...). 
* most recent examples: 
    * https://github.com/caponetto/vscode-diff-viewer/blob/main/documentation/original.png
    * https://github.com/caponetto/vscode-diff-viewer/blob/main/documentation/demo.png

it is not related to the resource size, and my internet speed is not _sooo_ slow either. it easily stays from 300KBps to 600KBps 2MiBps to 18 MiBps to even 40 MiBps in good signal strength areas.

- i even filed some issues to other repos - as i thought it was a breaking change from github regarding raw files: 
    * https://github.com/mwouts/jupytext/issues/1062
    * https://github.com/nhawkes/vscode-selection-overtype/issues/1#issuecomment-1704024774
- since i thought it was from github side, so i kept ignoring it - hoping it would somehow get fixed. the repo authors where i filled issues also said they can see the images, but I didn't understand how could it be. so, i ignored again.

but of late it become unbearable when browsing extensions in vscodium, so i thought to come here (🔍 term: [raw files](https://github.com/orgs/community/discussions?discussions_q=is%3Aopen+raw+files))

i have a hunch that it is limited to this geographical area only. 

here's the network response shown in chromium devtools:

![image](https://github.com/community/community/assets/19423063/0e5aebcb-e3b4-4c54-8f99-099e791c0bb7)


screenshot for images not shown when browsing extensions (open-vsx.com) via vscodium:
link: https://open-vsx.org/extension/byi8220/indented-block-highlighting

![image](https://github.com/community/community/assets/19423063/3d5b19ba-c286-4fe3-bf9e-5bf6182c8988)

**Reply by goyalyashpal** [reply]
same issue also discussed at: https://github.com/orgs/community/discussions/69838#discussioncomment-7286770

this issue hits the problem heads on: https://github.com/orgs/community/discussions/42655

Users from India using Jio as the mobile phone service carrier (ISP)

**Reply by goyalyashpal** [reply]
@ queenofcorgis sorry to ping, but i will urge you to report this issue to the domain side of github. due to how widely github repos are used all across the other sites too, this issue is crippling lots and lots of use cases.

**Reply by israel-oye** [reply]
> same issue also discussed at: https://github.com/orgs/community/discussions/69838#discussioncomment-7286770
> 
> this issue hits the problem heads on: https://github.com/orgs/community/discussions/42655
> 
> Users from India using Jio as the mobile phone service carrier (ISP)

The same issue is happening with Airtel users in Nigeria. Is it still the same with Jio or it has been resolved?

**Reply by goyalyashpal** [reply]
> Is it still the same with Jio or it has been resolved?
> \- _@ israel-oye at https://github.com/orgs/community/discussions/77343#discussioncomment-9806369_

i guess it was solved

**Comment by goyalyashpal** [comment]
@Yuheshpandian 

please stop spamming by creating new threads for each of ur comment. either edit ur older comment, or keep the related comments under same thread.

> can you mark mine as a answer then it would help me
> \- https://github.com/orgs/community/discussions/77343#discussioncomment-7718152

this is not a competition site to gain "selected answer" points.

**Reply by ghost** [reply]
Okay , thank you for the suggestions and corrections you had adviced me. I am a newbie so I don't know about that. I will sure change it.

**Comment by shakilmith** [comment]
Hi, I am facing issue regarding my hosted images at GitHub pages. It seems, this image
 https://docs.javaondemand.com/frontend/angular/angular-built-in-directives/ng-for-directive/image1.png  

works fine that is hosted at GitHub pages, but it doesn't load via img tag like <img src="https://docs.javaondemand.com/frontend/angular/angular-built-in-directives/ng-for-directive/image1.png" alt="demo_image" /> but, unfortunately it works on markdown syntax, that's ridiculous! Please, if you have any particular suggestion, kindly inform me.

**Comment by MohanReddy-235** [comment]
![Screenshot 2024-10-07 121419](https://github.com/user-attachments/assets/438ae744-206d-4f1b-9e0f-952913a72e90)
i am trying all ways but the images cannot be loading on github please any one knows to solve
